### PR TITLE
Support Mac OS / Darwin

### PR DESCRIPTION
**Pull request by uvwxy**
This is a quick hack to support Mac OS / darwin, as well as selecting the camera via index (macos only)

### CODE REVIEW

File: livecam.js [hunk 1]
```diff
@@ -179,6 +187,10 @@ function GstLiveCamServer(config) {
 			gst_video_src = 'ksvideosrc ! decodebin';
 		else if (OS.platform() == 'linux')
 			gst_video_src = 'v4l2src ! decodebin';
+		else if (OS.platform() == 'darwin')
+			gst_video_src = 'avfvideosrc';
```
Comment (3p3r) [inline livecam.js:191]: Nice! I never knew `avfvideosrc` existed!

**Comment by sirtawast** [comment]
Nice work @uvwxy, LGTM! After completing my PR I noticed this one, with better support for cameras using index. Maybe include the readme I improved? https://github.com/sepehr-laal/livecam/pull/4/commits/8a2dbe6dac53f847a66bc0061ee2be33fe9e0716#diff-04c6e90faac2675aa89e2176d2eec7d8

Could you @sepehr-laal merge this to master?

**Comment by 3p3r** [comment]
@sampohaavisto I would merge @uvwxy 's PR, but I do not own a Mac to test my comments above. Specifically, `avfvideosrc` and whether if it needs to be linked to a `decodebin` or not. On Windows and Linux it looks like so. Could you please test if it works with a `decodebin`? If so, I would take your word and merge this into master.

**Review by sirtawast — APPROVED** [review]
Device index 0 seems to be a test video stream. Device index 1 is the only index I got actual camera input from.

Tested with:
1. Main MBP cam (Retina, 13-inch, Early 2015)
2. External Apple display cam (connected to Lightning)
3. Logitech C922 webcam (only the USB near Lightning ports worked)

No other indices would work. Only one camera at a time would work.

Works with or without decodebin piping.

**Comment by sirtawast** [comment]
👏  @uvwxy @sepehr-laal

**Pull request merged**